pysus/online_data/__init__.py

```diff
@@ -68,6 +68,12 @@ def cache_contents():
 def parquets_to_dataframe(
     parquet_dir: str(PosixPath), clean_after_read=False
 ) -> pd.DataFrame:
+    """ 
+    Receives a parquet directory path and returns it as a
+    dataframe, trying to clean white spaces and convert to
+    the properly data types. Can read only one parquet dir
```

Comment:
replace 'properly' by 'correct'

Reply:
About this function, the only dtype that objects converts to is the String type, some white spaces in the values can be also found, e.g. `1020         `, that I think are preventing pandas to converts  to the correct dtypes

Reply:
After converting an object to string, apply a `strip` to it before trying the next conversion:
```python
d = "1020      "
d_converted = int(str(s).strip())
```